src/librustdoc/html/templates/item_info.html

```diff
@@ -0,0 +1,7 @@
+{%- if !items.is_empty() -%}
+<span class="item-info">
+{%- for item in items -%}
+{{item|safe}}{#- -#}
+{%- endfor -%}
+</span>
+{%- endif -%}
```

Comment:
From STYLE.md:

> We indent by four spaces after opening an HTML tag _or_ a Jinja tag.

Both this and the short_item_info.html template need some indentation tweaking.